roles/open_notificaties_k8s/templates/redis.yml.j2

```diff
@@ -47,3 +47,12 @@ spec:
         livenessProbe:
           tcpSocket:
             port: 6379
+        volumeMounts:
+        - mountPath: /data
+          name: data
+      volumes:
+      - emptyDir: {}
+        name: data
+
+      serviceAccountName: "{{ openzaak_service_account| default('default') }}"
+      securityContext: {{ opennotificaties_pod_security_context }}            
```

Comment:
It's not the intention to use redis snapshots or append-only, so probably we need to check the redis config and disable persistence completely.

Either way - we don't want to be managing volumes for redis containers, redis is 100% meant to be in-memory and serves as a cache server.

Reply:
Removed.

Reply:
As discussed, snapshots are now disabled by deploying redis.conf.